src/charactersheet/viewmodels/character/other_stats/index.js

```diff
@@ -1,4 +1,6 @@
+// eslint-disable-next-line no-unused-vars
 import { OtherStatsFormViewModel } from './form';
+// eslint-disable-next-line no-unused-vars

```

Comment:
Why have various unused imports had this line to ignore linting? Are the imports actually used somewhere?

Reply:
It looks like the view models need to be imported for Knockout to use them in the templates.